.github/workflows/componet-test.yml

```
@@ -45,14 +45,15 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Build Merge Two Streams example image and run component test
-        uses: trifork/cheetah-platform-examples/.github/actions/component-test@main
+        uses: trifork/cheetah-platform-examples/.github/actions/component-test@patch/fix-mergetwostreams-cp
```

```suggestion
        uses: .github/actions/component-test
```

Just make them relative